apps/version.c

```diff
@@ -126,9 +123,6 @@ int version_main(int argc, char **argv)
     if (options) {
         printf("options: ");
         printf(" %s", BN_options());
-#ifndef OPENSSL_NO_DES
-        printf(" %s", DES_options());
-#endif
```

Comment:
It's only deprecated, not removed

Reply:
All the other deprecations removed the option printing....